webapp/src/test/java/org/opentestsystem/rdw/admin/service/impl/DefaultCsvValidationServiceTest.java

```diff
@@ -74,6 +74,18 @@ public void itShouldNotValidateMissingHeaders() {
     }
 
     @Test
+    public void itShouldNotValidateAdditionalHeadersPastValidHeaders() {
+        final List<String> additionalHeaders = newArrayList(RequiredHeaders);
+        additionalHeaders.add("extra_header_at_end");
+
+        final CsvValidationFailure failure = service.validateHeaders(additionalHeaders).get(0);
+
+        assertThat(failure.getRow()).isEqualTo(0);
+        assertThat(failure.getMessage()).contains("Additional headers");
+        assertThat(failure.getMessage()).contains(Joiner.on(",").join(RequiredHeaders));
+    }
+
```

Comment:
Can you add a test where the found headers are the same as the required headers except for the missing last one?
```
    final List<String> headers = newArrayList(RequiredHeaders);
    headers.remove(headers.size()-1);
    ...
```